code-challenges.test.js

```diff
@@ -23,24 +28,115 @@ const people = [
+
+
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
 // a) Create a test with an expect statement using the variables provided.
 
+describe("mix2floor3", () => {
+  it("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.", () => {
+    expect(mix2div3(hodgepodge1)).toEqual([ 2, 0, -1, 0 ]);
+    expect(mix2div3(hodgepodge2)).toEqual([ 2, 1, -1 ]);
+
+  });
+});
+
 const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
 // Expected output: [ 2, 0, -1, 0 ]
 const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
 // b) Create the function that makes the test pass.
 
+// pseudo
+// input: mixed data array
+// output: remainders of the numbers divided by 3
+// - create a functions (mix2floor3) that takes and array of mix data 
+// - first .filter() the array to get only the typof 'number' from the array and have it go in to a varible (onlyNums)
+// - them map over the filter array of only numbers (onlyNums) to have all number by floor (%) by 3 and have it go into a varible (floor3)
+// - lastly have the function return the varible with the numbers being floor by 3 (floor3)
+
+
+
+const mix2floor3 = ( array ) =>{
+  let onlyNums = array.filter(
+    value => typeof value === 'number'
+    )
+  let floor3 = onlyNums.map(
+    val => val % 3
+    )
+  return floor3
+}
+
+  
+// console.log(mix2div3(hodgepodge1));
+// ---> [ 2, 0, -1, 0 ]
+// console.log(mix2div3(hodgepodge2));
+// ---> [ 2, 1, -1 ]
+
+
+
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
+describe("cubedThenSum", () => {
+  it("takes in an array of numbers and returns the sum of all the numbers cubed.", () => {
+    expect(cubedThenSum(cubeAndSum1)).toEqual(99);
+    expect(cubedThenSum(cubeAndSum2)).toEqual(1125);
+
+  });
+});
+
 const cubeAndSum1 = [2, 3, 4]
 // Expected output: 99
 const cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
 // b) Create the function that makes the test pass.
+
+// Pseudo
+// input: numbers array
+// Output: the sum of the numbers in the array after being cubed.
+// - First, lets .map() the array of numbers and have each number be cubed, lets store it in a varible (cubed).
+// - then, lets sum all the numbers in the array we cubed (cubed) with .reduce() wich would take the paramters of an accumulator(acc) and the curren number(curr) and have those two add with eachother, and lets store it in a varible (sum). **note: what this does is add the first number to the next hold that value as an accumulator and add the next number in the array until there is no more numbers in the array** 
```

Comment:
Love the mention of parameters in reduce!! Since you are using current number (curr), I am curious if you could refactor and take care of the cubed logic within the reduce method. Just a thought if you are up for a challenge.